sdk/servicebus/azure-servicebus/azure/servicebus/management/_management_client.py

```diff
@@ -603,11 +594,11 @@ def get_subscription_runtime_properties(self, topic, subscription_name, **kwargs
             entry.title, entry.content.subscription_description)
         return subscription
 
-    def create_subscription(self, topic, name, **kwargs):
-        # type: (Union[str, TopicProperties], str, Any) -> SubscriptionProperties
+    def create_subscription(self, topic_name, name, **kwargs):
```

Comment:
Maybe now that we've suffixed all the others, should we update `name` to `subscription_name`?

Reply:
(I found myself liking the pattern of prefix "<entity>_name" for clarity. But if we want the change, create_queue, create_topic, create_rule also need to be updated)

Reply:
I like and agree with this change, but both because of coming-to-wire for CC and, less-lazily-and-more-professionally, to keep this PR scope constrained, do folks mind if I put that as an issue for our next sprint?

Reply:
OOB folks concurred this is fine, [issue created](https://github.com/Azure/azure-sdk-for-python/issues/14225) for nov milestone.